packages/send-and-read-emails/client/react/src/SendEmails.js

```diff
@@ -0,0 +1,75 @@
+import { useNylas } from '@nylas/nylas-react';
+import { useState } from 'react';
+import { styles } from './styles';
+
+function SendEmails({ userId }) {
+  const nylas = useNylas();
+
+  const [to, setTo] = useState('');
+  const [body, setBody] = useState('');
+
+  const sendEmail = async ({ userId, to, body }) => {
```

Comment:
should there be a check for `userId`?

Reply:
Check already done https://github.com/nylas/use-cases/pull/29/files#diff-6e0891e8978a21f4cd4ce568daabcdf118edaf6d39f85ea706af713c3bbed012R35